src/app/sitemap.ts

```diff
@@ -0,0 +1,9 @@
+import { CATEGORIES, SITE_URL } from '@/constants';
+import { MetadataRoute } from 'next';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+    return CATEGORIES.map((category) => ({
```

Comment:
You also need to combine this with a list of static pages:
- Credits
- Jobs
- Home page

Is there any way to get a list of routes/pages dynamically?

Reply:
Ok let me research about it.